auto_nag/db.py

```diff
@@ -0,0 +1,383 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import calendar
+import csv
+import dateutil.parser
+from filelock import FileLock
+import json
+from libmozdata import utils as lmdutils
+import os
+import six
+from sqlalchemy.ext.declarative import declarative_base
+from sqlalchemy import create_engine
+from sqlalchemy.orm import relationship, sessionmaker
+from sqlalchemy.orm.exc import NoResultFound
+from sqlalchemy import Column, ForeignKey, Integer, String
+
+from auto_nag import logger, utils
+from auto_nag.history import History
+
+
+Base = declarative_base()
+lock_path = utils.get_config('common', 'lock')
+db_url = utils.get_config('common', 'database')
+engine = create_engine(db_url)
+DBSession = sessionmaker(bind=engine)
+Base.metadata.bind = engine
+session = DBSession()
+
+
+def init():
```

Comment:
Is this still needed ? I don't find any ref to this function

Reply:
yep, just to init the db with data from bugzilla